### ISSUE TITLE
For 5 days persist issue with "Failed to generate response; topup: false"

### ISSUE THREAD

**Issue by SKYlOVEBEST** [OnlyRizzAI/public-tracker#9]
Hello admins: For 5 days persist issue with "Failed to generate response; topup: false". Hope You can solve this very soon also for other users. Thanks. 
![can not chat](https://github.com/OnlyRizzAI/public-tracker/assets/173850130/0db6699f-ab56-497e-a320-232efa7808ca)

**Comment by Mitch089089** [comment]
Same problem since 5 Days, and i also hope you can solve it asap cause it´s lasts too long for now. Thanks

**Comment by Natsugo** [comment]
Yeah I've been having the exact same issue, it started right after I paid for the subscription, and I haven't been able to send messages at all ever since.
